Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.yml

```diff
@@ -74,6 +74,15 @@ configuration:
   required: false
   additionalinfo: Limit on the number of incidents retrieved in a single fetch.
   section: Collect
+- display: Page size
+  name: page_size
+  type: 0
+  required: false
+  section: Collect
+  defaultvalue: '50000'
+  additionalinfo: The number of events to fetch per request - the maximum is 600k, raise this parameter in case you're getting aggregated delays.
```

Comment:
The number of events to fetch per request.  Increase the number if you're getting aggregated delays. The maximum is  600,000.